### ISSUE TITLE
Keras implementation of the loss function

### ISSUE THREAD

**Issue by nixingyang** [tjddus9597/Proxy-Anchor-CVPR2020#5]
I have implemented the [loss function](https://github.com/nixingyang/Proxy-Anchor-Loss) using Keras. Hope the community will find it helpful :-)

**Comment by sung-yeon-kim** [comment]
Sorry for the late reply. Thank you so much for your great implementation.
I will add this implementation to 'Other Implementations' tab.
